### ISSUE TITLE
Load embeddings: vocab

### ISSUE THREAD

**Issue by axnedergaard** [coree/ChiNet#16]
The load_embeddings() function in model.py needs access to our vocab (used in preprocessor.py?)

**Comment by niladell** [comment]
About that, sure, we have to make sure that the words on our training set are on the embedding. But I don't see why we would want to restrict ourselves to a smaller vocab_size, when in the test set there may be many novel words (specially knowing I wrote some of them with really obscure sentences). 

Sure, we may want to constrict that for comparing with the original work, I can see that, but for the final submission I'd go with the whole vocabulary.

Having said that I implement it for now with limited vocabulary, but having a big "!!!" for the final version.
